components/AppBar/src/MDCAppBar.m

```diff
@@ -220,6 +220,7 @@ - (UIBarButtonItem *)backButtonItem {
                                                         action:@selector(didTapBackButton:)];
   }
   backBarButtonItem.accessibilityIdentifier = @"back_bar_button";
+  backBarButtonItem.accessibilityLabel = NSLocalizedString(@"Back", @"Back");
```

Comment:
Yep I only published this as it is a blocker for a project that needed a temporary fix and this was a way for me to display the fix. I've marked it with a "isPlanningChanges" label so nobody merged it or reviewed it.